libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -69,6 +70,7 @@ var subsystems = subsystemSet{
 const (
 	testScopeWait = 4
 	testSliceWait = 4
+	cgroupUnified = "/sys/fs/cgroup/unified"
```

Comment:
Shouldn't we will look for the cgroups mountpoint and append `unified` to it in cause cgroups is not mounted under `/sys/fs`?

Reply:
Please notice that this is `systemd` specific, in hybrid mode the cgroup-v2 is mounted on `/sys/fs/cgroup/unified` as stated here: https://systemd.io/CGROUP_DELEGATION.html#three-different-tree-setups-.

Reply:
Not to mention that these days, `/dev/cgroup` is dead and everyone mounts it to `/sys/fs/cgroup`.

Reply:
I think this variable should be named `cgroupHybridUnified` to avoid confusion

Reply:
@AkihiroSuda I just renamed the variable. Do you have other feedback about this PR? Thanks.